rotkehlchen/exchanges/binance.py

```diff
@@ -794,7 +794,7 @@ def query_balances(self) -> ExchangeQueryBalances:
             self.first_connection()
             returned_balances: DefaultDict[Asset, Balance] = defaultdict(Balance)
             returned_balances = self._query_spot_balances(returned_balances)
-            if self.name != str(Location.BINANCEUS):
+            if self.location != Location.BINANCEUS:
```

Comment:
no need to do this, this is already done in bugfixes and may even conflict during merging bugfixes to develop, so please don't